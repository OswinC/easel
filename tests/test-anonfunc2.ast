function int foo(function int (int, int) fp, int x, int y)
{
x = x * 10;
y = y + 1;
return fp(x, y);
}

int i, j, a[3][3];
for (i = 0 ; i < a.size ; i++) {
for (j = 0 ; j < a[0].size ; j++) {
print(foo(function int (int x, int y)
{
return a[i][j] = 100 + x + y;
}, i, j));
}
}
