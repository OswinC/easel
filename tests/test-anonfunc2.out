101
102
103
111
112
113
121
122
123
